flexmeasures/data/scripts/cli_tasks/data_add.py

```diff
@@ -94,17 +99,70 @@ def new_user(username: str, email: str, roles: List[str], timezone: str):
     required=True,
     help="timezone as string, e.g. 'UTC' or 'Europe/Amsterdam'",
 )
+@click.option(
+    "--generic-asset-id",
+    required=True,
+    type=int,
+    help="Generic asset to assign this sensor to",
+)
 def add_sensor(**args):
     """Add a sensor."""
     check_timezone(args["timezone"])
     check_errors(SensorSchema().validate(args))
     args["event_resolution"] = timedelta(minutes=args["event_resolution"])
     sensor = Sensor(**args)
-    app.db.session.add(sensor)
-    app.db.session.commit()
+    db.session.add(sensor)
+    db.session.commit()
     print(f"Successfully created sensor with ID {sensor.id}")
-    # TODO: uncomment when #66 has landed
-    # print(f"You can access it at its entity address {sensor.entity_address}")
+    print(f"You can access it at its entity address {sensor.entity_address}")
+
+
+@fm_dev_add_data.command("generic-asset-type")
+@with_appcontext
+@click.option("--name", required=True)
+@click.option(
+    "--hover-label",
+    type=str,
+    help="Label visible when hovering over the name in the UI.\n"
+    "Useful to explain acronyms, for example.",
+)
+def add_generic_asset_type(**args):
+    """Add a generic asset type."""
+    check_errors(GenericAssetTypeSchema().validate(args))
+    generic_asset_type = GenericAssetType(**args)
+    db.session.add(generic_asset_type)
+    db.session.commit()
+    print(f"Successfully created generic asset type with ID {generic_asset_type.id}")
+    print("You can now assign generic assets to it")
+
+
+@fm_dev_add_data.command("generic-asset")
+@with_appcontext
+@click.option("--name", required=True)
+@click.option(
+    "--latitude",
+    type=float,
+    help="Latitude of the asset's location",
+)
+@click.option(
+    "--longitude",
+    type=float,
+    help="Longitude of the asset's location",
+)
+@click.option(
+    "--generic-asset-type-id",
+    required=True,
+    type=int,
+    help="Generic asset type to assign this sensor to",
+)
+def add_generic_asset(**args):
```

Comment:
Wouldn't it be useful to add an owner ID here as well? That would also require to add that field in the schema.

Reply:
You resolved this in https://github.com/SeitaBV/flexmeasures/pull/159#r693786541.